src/components/new-members/new-members.module.scss

```diff
@@ -34,9 +34,13 @@
   height: 160px;
   overflow: hidden;
   margin: auto;
-  border: 0;
+  border: none;
```

Comment:
Why was this change made?

Reply:
Both are valid but shouldn't have changed , I have reverted back the `border: 0`